bin/zopen-help2man

```diff
@@ -67,23 +67,20 @@ for tooldir in ${tooldirs}; do
   for tool in $tooldir/zopen*; do
     basetool=$(basename ${tool})
     help2man ${help2manopts} "${tool}" >"${mandir}/${basetool}.1"
-    if  $( "${tool}" --help >/dev/null 2>&1 ) ; then
+    if $("${tool}" --help >/dev/null 2>&1); then
       # Tool has implemented --help
-      if  $( "${tool}" --version >/dev/null 2>&1 ) ; then
+      if $("${tool}" --version >/dev/null 2>&1); then
```

Comment:
i kinda like a space here - but if others don't, i'm ok to change... e.g. i prefer `$( <cmd> )` rather than `$(<cmd>)`